assets/lokomotive-kubernetes/bootkube/resources/charts/kube-apiserver/templates/kube-apiserver.yaml

```diff
@@ -221,9 +239,11 @@ spec:
 
           {{ end }}
           backend kube-apiserver
-            server 1 $ADDRESS:{{ template "port" . }} check
+            option httpchk GET /healthz HTTP/1.1\r\nHost:\ kube-apiserver\r\nAuthorization:\ Bearer\ {{ template "token" . }}
+            server 1 $ADDRESS:{{ template "port" . }} verify none check check-ssl
           EOF
           cat /run/kube-apiserver/haproxy.cfg
+          echo "{{ template "token" . }},nobody,nobody" > /run/kube-apiserver/token-auth-file
```

Comment:
I was trying to find info on this "nobody" user but couldn't find anything. Do you have a reference here? Or is it just a non-existent user and that means they don't have any privileges?

Reply:
It's just an arbitrary user name, which do not have any RBAC associated with. I wouldn't say it's "non-existent", as K8s users are not really stateful. If it has valid API token/cert, it appears in the system :smile: 

If you have better suggestions, I'm happy to adopt.

Reply:
that's fine then

Reply:
I'll add an inline comment about it for the time being.